wallet/src/components/BridgeComponent/index.jsx

```diff
@@ -205,8 +212,10 @@ const BridgeComponent = () => {
               throw new Error(err);
             });
           break;
-        default:
+        }
+        default: {
```

Comment:
Remove curly brackets here

Reply:
Yeah I think was a problem with eslint. I"ll remove and try push.

Reply:
If you don't want this brackets ... we'll need to the eslint rule for that.

https://eslint.org/docs/rules/no-case-declarations